tests/system/test_0026_test_config.py

```diff
@@ -16,7 +16,7 @@ def test_ok_config(self):
         """
         self.render_config_template()
         exit_code = self.run_packetbeat(pcap="http_post.pcap",
-                                        extra_args=["-test"])
+                                        extra_args=["-configtest"])
```

Comment:
Oops, I should have done a file search for `-test` when I made the change to libbeat.